app/client/components/funding/wyreBankFunding.jsx

```diff
@@ -9,7 +9,7 @@ import {
 } from "../../reducers/wyreReducer";
 import LoadingSpinner from "../loadingSpinner.jsx";
 import AsyncButton from "../AsyncButton.jsx";
-import { loadBusinessProfile } from "../../reducers/businessVerificationReducer";
+import { loadBusinessProfile } from "../../reducers/businessVerification/reducers";
```

Comment:
lol we should probably rename the /reducers/ directory something like /redux/ so we don't have `/reducers/foo/reducers`